crates/topos-sequencer-subnet-runtime/src/proxy.rs

```diff
@@ -193,7 +194,7 @@ impl SubnetRuntimeProxy {
                             runtime_proxy.clone(),
                             &mut subnet_listener,
                             certification.clone(),
-                            next_block_number,
+                            next_block_number as u64,
```

Comment:
Why `u64` here and not `u128` like above (instead of signed)?

Reply:
Above is i128, to indicate with -1 that we do not have any block so far. API for subnet client expects u64 in general.